ci-operator/step-registry/enable-qe-catalogsource/logging/enable-qe-catalogsource-logging-ref.yaml

```diff
@@ -9,11 +9,27 @@ ref:
       memory: 100Mi
   env:
   - name: LOGGING_INDEX_IMAGE
-    default: "quay.io/openshift-qe-optional-operators/ocp4-index:latest"
-    documentation: The Logging QE CatalogSource index image.
+    default: ""
+    documentation: |-
+      The Openshift Logging catalogSource image. the available image is quay.io/openshift-qe-optional-operators/aosqe-index:xxx,brew.registry.redhat.io/rh-osbs/iib:xxx or registry.xxx.redhat.io/redhat/redhat-operator-index:xxx
+  - name: CLO_INDEX_IMAGE
+    default: ""
+    documentation: "The cluster-logging catalogSource image"
+  - name: LO_INDEX_IMAGE
+    default: ""
+    documentation: "The loki-operator catalogSource image"
+  - name: EO_INDEX_IMAGE
+    default: ""
+    documentation: "The elasticsearch-operator catalogSource image"
+  - name: LOG_VERSION
+    default: ""
+    documentation: "The Logging version will be  deployed. For example: 6.3"
+  - name: MIRROR_IMAGES
```

Comment:
Please remove the unused env var.

Reply:
Fixed

Reply:
`MIRROR_IMAGES` is not used in the script.